tools/build_pak_files.py

```diff
@@ -0,0 +1,100 @@
+#
+# Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+#
+
+import argparse
+import glob
+import os
+import subprocess
+import sys
+
+
+if __name__ == '__main__':
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--unreal_binaries_dir", required=True)
+    parser.add_argument("--unreal_project_dir", required=True)
+    parser.add_argument("--scene_id", required=True)
+    parser.add_argument("--platforms", nargs="*", required=True, help="Supported platform values are Windows, Linux, and Mac")
+    parser.add_argument("--output_dir", required=True)
+    args = parser.parse_args()
+
+    os.makedirs(args.output_dir, exist_ok=True)
+    
+    assert os.path.exists(args.unreal_binaries_dir)
+    unreal_editor_exe = os.path.realpath(os.path.join(args.unreal_binaries_dir, "Win64", "UE4Editor.exe"))
+    unreal_pak_exe    = os.path.realpath(os.path.join(args.unreal_binaries_dir, "Win64", "UnrealPak.exe"))
+    
+    assert os.path.exists(args.unreal_project_dir)
+    _, project = os.path.split(args.unreal_project_dir)
+    uproject   = os.path.realpath(os.path.join(args.unreal_project_dir, project + ".uproject"))
+    
+    for platform in args.platforms:
+
+        cmd = [ 
+            unreal_editor_exe,
+            uproject,
+            "-run=Cook",
+            "-TargetPlatform=" + platform + "NoEditor",
+            "-fileopenlog",
+            "-ddc=InstalledDerivedDataBackendGraph",
+            "-unversioned",
+            "-stdout",
+            "-FullStdOutLogOutput",
+            "-CrashForUAT",
+            "-unattended",
+            "-NoLogTimes",
+            "-UTF8Output"
+        ]
+        print(f"[SPEAR | build_pak_files.py] Executing: {' '.join(cmd)}")
+        cmd_result = subprocess.run(cmd, stdout=sys.stdout.buffer, stderr=sys.stdout.buffer)
```

Comment:
I don't think setting stdout= and stderr= like this has any effect because you're just setting it to the default. Are you sure this has an effect?

Reply:
You are right, they did not have any effect. Removed them.